github_stargazers/github_stargazers.py

```diff
@@ -7,53 +7,37 @@
 from github_stargazers.github import UsernameRepositoryError, TooManyRequestsHttpError, HTTPError
 
 
+class OutputPrintable(object):
```

Comment:
Since this class is needed and used only in this module, it should be private to this module only, as we did with `_Command`, by renaming it as `_OutputPrintable`. 
Also by doing this we don't need new unit-tests for it because any further potential issues will be exposed by the tests from `test_github_stargazers.py`.